docs/src/pages/release-notes/changelog.md

```diff
@@ -30,6 +30,7 @@ The minor version will be incremented upon a breaking change and the patch versi
 - idl: Remove `anchor-syn` dependency ([#3030](https://github.com/coral-xyz/anchor/pull/3030)).
 - lang: Add `const` of program ID to `declare_id!` and `declare_program!` ([#3019](https://github.com/coral-xyz/anchor/pull/3019)).
 - idl: Add separate spec crate ([#3036](https://github.com/coral-xyz/anchor/pull/3036)).
+- cli: Include recommended solana args by default and add new --max-retries ([#3354](https://github.com/coral-xyz/anchor/pull/3354)).
```

Comment:
> Sorry for the wait, fixed the clippy lints and updated the changelog

Thanks but this is the incorrect one to update (v0.30.1 is already out, this will be included in v0.31.0). Please revert this change and update it in the place that was linked in https://github.com/coral-xyz/anchor/pull/3354#pullrequestreview-2442605087